src/components/AutoComplete/AutoComplete.tsx

```
@@ -0,0 +1,207 @@
+                >
+                  {highlightMatch(item, inputValue)}
+                </li>
+              ))}
+            </ul>
+          ) : (
+            <div className="p-2">
+              <button
+
+                onClick={handleAddNew}
+                className="typography-body-s text-secondary-indigo w-full flex items-center py-12 px-16 cursor-pointer justify-start gap-2"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)